segmentation/utils/visualize.py

```diff
@@ -1,20 +1,78 @@
 import pandas as pd
 import numpy as np
-import matplotlib
+import matplotlib.pyplot as plt
 import seaborn as sns
 
 
 def visualize_z_scores(z, pheno_titles):
     """Plots the z scores between all phenotypes as a clustermap.
-
     Args:
         z: array of z scores of shape (pheno_num x pheno_num)
         pheno_titles: list of all the names of the phenotypes"""
-    # visualize
     # Replace the NA's and inf values with 0s
     z[np.isnan(z)] = 0
     z[np.isinf(z)] = 0
     # Assign numpy values respective phenotype labels
     zplot = pd.DataFrame(z, columns=pheno_titles, index=pheno_titles)
     sns.set(font_scale=.7)
     sns.clustermap(zplot, figsize=(8, 8), cmap="vlag")
+
+
+def swap_columns(df, c1, c2):
+    df['temp'] = df[c1]
+    df[c1] = df[c2]
+    df[c2] = df['temp']
+    df.drop(columns=['temp'], inplace=True)
+    return df
+
+
+def getSortedDf(df, patient_col_name, population_col_name, isNormalized=False):
```

Comment:
Lets change this from CamelCase to snake_case naming convention, which we use in python

Reply:
Also add a docstring